ugvc/mrd/mrd_utils.py

```diff
@@ -92,6 +92,16 @@ def _safe_tabix_index(vcf_file: str):
     return vcf_file
 
 
+def _validate_info(x):
```

Comment:
First check if the input is boolean and if so return it, then checks whether it's a string before doing string operations, and please add a docstring and explain what this is for.